### PR TITLE
feat: setup timeline buitlin widget

### PR DESCRIPTION
**Pull request by keiya01**
# Overview

## What I've done

I added config for builtin timeline widget in manifest.yml.

## How I tested

- run reearth-backend
- run reearth-web with https://github.com/reearth/reearth-web/pull/285
- then you can add timeline widget from side panel
- and you can operate timeline component

### CODE REVIEW

File: pkg/builtin/manifest.yml
Comment (KaWaite) [inline pkg/builtin/manifest.yml:1925]: This should be true, shouldn't it? Do we actually want multiple timelines??

File: pkg/builtin/manifest.yml
Comment (KaWaite) [inline pkg/builtin/manifest.yml:1924]: "A timeline widget that allows for viewing layers and data at specific points in time"

File: pkg/builtin/manifest.yml
Comment (keiya01) [reply pkg/builtin/manifest.yml]: This was fixed by [fix: enable singleOnly flag](https://github.com/reearth/reearth-backend/pull/161/commits/27996b0eaeebb67f482db8e2b9c26f859cf83479)

File: pkg/builtin/manifest.yml
Comment (keiya01) [reply pkg/builtin/manifest.yml]: This was fixed by [fix: improve description](https://github.com/reearth/reearth-backend/pull/161/commits/43413d48f2dbea9f6a2dd637770ace5a8fef3542).
@rot1024 Could you review Japanese translation?

**Review by KaWaite — APPROVED** [review]
Approved assuming Japanese is updated and this PR is updated with main branch before merging!

File: pkg/builtin/manifest_ja.yml
Comment (rot1024) [inline pkg/builtin/manifest_ja.yml:909]:
```suggestion
    description: 時系列データを表示するために、時刻を変更したり時間を再生したりすることができます。
```

**Review by rot1024 — APPROVED** [review]

**Review by HideBa — APPROVED** [review]
LGTM

**Pull request merged**
